pkg/server/testserver.go

```diff
@@ -691,6 +691,37 @@ func (ts *testServer) Activate(ctx context.Context) error {
 		return err
 	}
 
+	// If the test is indirectly dependent on rangefeed latency, but not
+	// particularly interested in observing the effects of the default
+	// rangefeed configuration, make the test faster.
```

Comment:
In one way or another, every test uses rangefeeds.  I wonder what the impact of these settings on some of the unrelated tests (not touched by this PR).  How much of a startup speed this brings (if any)?  If there is impact, perhaps, this is just something that should be done unconditionally?